js/index.js

```diff
@@ -0,0 +1,202 @@
+           FI_Rate > 0.131 ? '#ff6600' :
+           FI_Rate > 0.123 ? '#ff9b00' :
+           FI_Rate > 0.114 ? '#ffcb00' :
+           '#fff700';
+      };
+      const borderColor = feature.properties.County === 'Philadelphia County, Pennsylvania' ? 'blue' : 'black';
+      const borderWeight = feature.properties.County === 'Philadelphia County, Pennsylvania' ? 3 : 2;
+      return {
+        fillColor: getColor(feature.properties.FI_Rate),
+        weight: borderWeight,
+        opacity: 1,
+        color: borderColor,
+        fillOpacity: 0.7,
+      };
+    },
+  },
+  '2-Poverty-slide': {
+    style: (feature) => {
+      const getColor = (PercPoor) => {
+        return PercPoor > 0.33816 ? '#1527c8' :
+        PercPoor > 0.23302 ? '#425dcf' :
+        PercPoor > 0.15836 ? '##6f93d7' :
+        PercPoor > 0.09148 ? '#9bc9de':
+             '#c8ffe5';
+      };
+      return {
+        fillColor: getColor(feature.properties.PercPoor),
+        weight: 2,
+        opacity: 1,
+        color: 'black',
+        fillOpacity: 0.7,
+      };
+    },
+    onEachFeature: (feature, layer) => {
+      // Calculate the poverty percentage and round it to two decimal places
+      const povertyPercentage = (feature.properties.PercPoor * 100).toFixed(2);
+      // Bind the tooltip to the polygon
+      layer.bindTooltip(`Poverty: ${povertyPercentage} %`, {
+        permanent: false, // Tooltip appears on hover
+        direction: 'top', // Tooltip position
+      });
+    },
+  },
+  '3-CB-slide': {
+    style: (feature) => {
+      const getColor = (PercCB) => {
+        return PercCB > 0.62828 ? '#1527c8' :
+        PercCB > 0.53602 ? '#425dcf' :
+        PercCB > 0.44682 ? '##6f93d7' :
+        PercCB > 0.34192 ? '#9bc9de':
+             '#c8ffe5';
+      };
+      return {
+        fillColor: getColor(feature.properties.PercCB),
+        weight: 2,
+        opacity: 1,
+        color: 'black',
+        fillOpacity: 0.7,
+      };
+    },
+    onEachFeature: (feature, layer) => {
+      const costburdenPercentage = (feature.properties.PercCB * 100).toFixed(2);
+      layer.bindTooltip(`Cost-Burdened: ${costburdenPercentage} %`, {
+        permanent: false,
+        direction: 'top',
+      });
+    },
+  },
+  '4-Access-slide': {
+    style: (feature) => {
+      const getColor = (TOTAL_HPSS) => {
+        return TOTAL_HPSS > 5 ? '#3b0d52' :
+        TOTAL_HPSS > 3.25 ? '#6a3979' :
+        TOTAL_HPSS > 2.25 ? '#9a66a3' :
+        TOTAL_HPSS > 1 ? '#cc96cf':
+             '#ffc9fd';
+      };
+      return {
+        fillColor: getColor(feature.properties.TOTAL_HPSS),
+        fillOpacity: 0.7,
+        weight: 0,
+        opactiy: 0,
+      };
+    },
+  },
+  '5-FS-slide': {
+    style: (feature) => {
+      const getColor = (PercFS) => {
+        return PercFS > 0.43724 ? '#145c49' :
+        PercFS > 0.28752 ? '#178556' :
+        PercFS > 0.18164 ? '#3faf56' :
+        PercFS > 0.07516 ? '#78d848':
+             '#bdff23';
+      };
```

Comment:
**praise:** I don't know whether it was intentional, but you kinda matched the colors on the SNAP EBT cards for PA, which is neat:

![image](https://github.com/user-attachments/assets/86497726-045d-44d3-98fb-bc4df3f912ac)

![image](https://github.com/user-attachments/assets/1e729a07-6436-4688-8be4-e58804ed01d0)